docs/software-platforms/aws/working-with-s3-buckets.md

```diff
@@ -40,4 +40,36 @@ You can then use `$OPENSCPCA_RESULTS_BUCKET` (or, `${OPENSCPCA_RESULTS_BUCKET}`)
 
 ## Syncing your results to S3
 
-_Coming up in the next PR._
+We have written a script to help you sync you results to your S3 bucket, stored in [`scripts/sync-results.py`](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/scripts/sync-results.py).
+You will need to use the [terminal](../general-tools/using-the-terminal.md) to run this script.
+
+For this script to you, you need to be [logged into the AWS account profile you use for contributing to OpenScPCA](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
+
+
+The simplest usage of this script, called from the `OpenScPCA-analysis` repository root folder, is:
```

Comment:
We should say somewhere prominently what the script does here: that it syncs the contents of the `results` and `plots` directories _only_ to S3, organized by module.

Note the sync: if there have been files removed from the local directory, it will delete them, unless told not to. (We could change this behavior: I debated which was "better")

Reply:
Hmmmm... I think I'd change this behavior (yes, I know I was the original reviewer). I'm now thinking about the situation where someone is switching between LSfR and their laptop. It'd be swell to be able to copy things to S3 from either local situation without worrying.

Reply:
Yeah, that makes sense. On naming... how does `--exact-sync` or something like that sound? `destructive-sync`? Or just `--with-delete`?